ChangeLog.md

```diff
@@ -1,14 +1,19 @@
-## Changes in 1.0.1
+# Changes in 1.0.2
+
+* CMake: add `CLAP_BUILD_TESTS` which enables the tests.<br/>
+  One test required a C17 compiler which is not available from Visual Studio 2019 and gcc9.
```

Comment:
Perhaps change this line to more specifically mention that this fixes a regression that broke compilation under Visual Studio 2019 and GCC9.